visualization/package.json

```diff
@@ -85,6 +84,7 @@
 		}
 	},
 	"dependencies": {
+		"@babel/polyfill": "^7.10.4",
```

Comment:
This module is [deprecated](https://babeljs.io/docs/en/babel-polyfill) and it's recommended to include core-js etc. directly instead.